scripts/initialize.sh

```diff
@@ -33,7 +36,11 @@ help()
    echo -e "\t--beneficiary: Staking beneficiary address"
    echo -e "\t--authorizer: Staking authorizer address"
    echo -e "\t--stake-amount: Staking amount"
-   echo -e "\t--authorization-amount: Authorization amount\n"
+   echo -e "\t--authorization-amount: Authorization amount"
+   echo -e "\t--tbtc-path: 'Local' tbtc project's path. A temporary folder with tbtc is created and removed"\
+                           "upon installation if the path is not provided"
```

Comment:
Why do we need a path for tbtc at all for initialization?

Reply:
Say you have `tbtc` project on your local machine. You run `install.sh --tbtc-path <your_local_tbtc_path>`. Now you need to do the same with `initialize.sh --tbtc-path <your_local_tbtc_path>` because all the contracts were deployed from there and all the artifacts are there.